src/helperFunctions/install.py

```diff
@@ -53,89 +74,135 @@ def run_shell_command_raise_on_return_code(command: str, error: str, add_output_
 
 
 def dnf_update_sources():
-    return run_shell_command_raise_on_return_code('sudo dnf update -y', 'Unable to update')
+    '''
+    Update package lists on Fedora / RedHat / Cent systems.
+    '''
+    return _run_shell_command_raise_on_return_code('sudo dnf update -y', 'Unable to update')
+
 
+def dnf_install_packages(*args: str):
```

Comment:
I don't think `args` is the best name for this variable. Maybe "packages"? (applies to other functions as well)